modules/util/css-builder/src/main/java/com/liferay/css/builder/maven/BuildCSSMojo.java

```diff
@@ -64,10 +68,31 @@ public void execute() throws MojoExecutionException {
 						_cssBuilderArgs.setImportDir(artifact.getFile());
 					}
 
+					artifactPresent = true;
+
 					break;
 				}
 			}
 
+			if (!artifactPresent) {
+				for (ComponentDependency componentDependency :
+						_pluginDescriptor.getDependencies()) {
+
+					String artifactId = componentDependency.getArtifactId();
+
+					if (artifactId.equals("com.liferay.frontend.css.common") &&
+						(_cssBuilderArgs.getImportDir() == null)) {
```

Comment:
Part of the conditional check can be moved before the `for` loop to optimize logic and readability.  The same pattern can be applied to `L63`.

```
if (!artifactPresent && (_cssBuilderArgs.getImportDir() != null)) {
```